### DISCUSSION TITLE
Show module content from REPL

### DISCUSSION THREAD

**Discussion by tech-cobber** [geldata/gel#1857] [Q&A] [answered]
Is there any way to list all the types, functions,... inside a specific module? Don't see anything like that in `\help` or docs

**Comment by elprans** [comment, marked as answer]
`DESCRIBE MODULE <module-name> AS SDL;` should do the trick.

**Reply by tech-cobber** [reply]
I tried on `std` and got this:

![image](https://user-images.githubusercontent.com/50849313/95027404-e1565e80-06a0-11eb-9571-7af2888441db.png)

**Reply by elprans** [reply]
Hm, `DESCRIBE` seems to be broken for `std` currently. Opened #1858 to track the fix.

`\list-scalar-types -s std::` (and similar commands, see `\help`) is another way to list things.

**Reply by tech-cobber** [reply]
Thank you! `DESCRIBE` works fine for other modules. I also tried list commands, but it didn't occur to me to add `::` after module name :)
